web/src/components/auth/user-profile.tsx

```
@@ -0,0 +1,85 @@
+  className?: string;
+}
+
+export function UserProfile({ className }: UserProfileProps) {
+  const { user, isLoading, error } = useUser();
+
+  if (isLoading) {
+    return (
+      <Card className={className}>
+        <div className="p-4">
+          <div className="animate-pulse">
+            <div className="h-4 bg-gray-200 rounded w-3/4 mb-2"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)